### PR TITLE
fix README.md

### PR DESCRIPTION
**Pull request by raa0121**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,8 +170,8 @@ You can change the directory to install servers by set `g:lsp_settings_servers_d
 | Vim               | vim-language-server                 |    Yes    |      Yes      |
 | Vala              | vala-language-server                |    No     |      No       |
 | Vue               | volar-server                        |    Yes    |      Yes      |
-| Vue               | vue-language-server                 |    Yes    |      Yes      |
-| V                 | vls                                 |    Yes    |      Yes      |
+| Vue               | vls                                 |    Yes    |      Yes      |
+| V                 | vls(vlang-vls)                      |    Yes    |      Yes      |
```
Comment (mattn) [inline README.md:174]: Thank you. Just replace to vlang-vls

Comment (mattn) [inline README.md:174]:
```suggestion
| V                 | vlang-vls                           |    Yes    |      Yes      |
```

Comment (raa0121) [reply README.md]: fix on 98b3aea

**Comment by mattn** [comment]
Thank you

**Pull request merged**